web/react/components/create_comment.jsx

```diff
@@ -205,6 +206,11 @@ module.exports = React.createClass({
             );
         }
 
+        var extraInfo = <MsgTyping channelId={this.props.channelId} parentId={this.props.rootId}  />;
+        if (this.state.messageText.split(' ').length > 1 && allowTextFormatting) {
```

Comment:
Is there some reason that we're not showing the formatting stuff until at least two words have been typed? It seems weird to have it pop in like that at a rather off time. I feel like it'd be better to just show it all the time and use styling to make it less visible.